tidb-configuration-file.md

```diff
@@ -355,6 +355,12 @@ Configuration items related to performance.
 - Default: `NO_PRIORITY`
 - Optional values: `NO_PRIORITY`, `LOW_PRIORITY`, `HIGH_PRIORITY` and `DELAYED`.
 
+### `nested-loop-join-cache-capacity`
+
++ The upper memory limit for the Least Recently Used (LRU) nested loop join cache, which indicates the upper memory threshold (in bytes).
```

Comment:
```suggestion
+ The maximum memory usage for the Least Recently Used (LRU) algorithm of the nested loop join cache (in bytes).
```
`which indicates` part seems to carry no more information than the main clause, so I suggest removing it. Please help confirm whether my suggestion is technically correct. (Is LRU is an algorithm?) @qw4990